src/server/server_state.h

```diff
@@ -282,7 +282,12 @@ class ServerState {  // public struct - to allow initialization.
   // Decommits 3 possible heaps according to the flags.
   // For decommit_glibcmalloc the heap is global for the process, for others it's specific only
   // for this thread.
-  enum { kDataHeap = 1, kBackingHeap = 2, kGlibcmalloc = 4 };
+  enum {
+    kDataHeap = 1,
+    kBackingHeap = 2,
+    kGlibcmalloc = 4,
+    kAll = kDataHeap | kBackingHeap | kGlibcmalloc
```

Comment:
Since this is an anonymous enum, I'd use a more verbose name. Maybe `kAllHeaps` or `kAllMemory`?
Alternatively we can name this enum (preferably use `enum class`) so that it will not be confusing.
Anyway, this is a nit so feel free to ignore.